views/sponsors.php

```diff
@@ -17,6 +17,8 @@
 
     <!-- CSS Pages -->
     <link rel="stylesheet" type="text/css" href="/public/css/style.css">
+        <link rel="stylesheet" type="text/css" href="/public/css/sponsors.css">
```

Comment:
Can you un-indent this so that it lines up with the <link> tags above and below it